frontend/src/scenes/error-tracking/ErrorTrackingScene.tsx

```diff
@@ -76,29 +77,21 @@ const CustomVolumeColumnHeader: QueryContextColumnTitleComponent = ({ columnName
 }
 
 const CustomGroupTitleColumn: QueryContextColumnComponent = (props) => {
-    const { value, record } = props
+    const { value, record } = props as { value: string; record: ErrorTrackingGroup }
```

Comment:
Happens because we use `unknown` in a type elsewhere

https://github.com/PostHog/posthog/blob/85381088775bee4007230bb611863cd819a7a04b/frontend/src/queries/types.ts#L46-L47

I should probably clean that up with generics but happy to work around it for now